docs/src/man/faq.md

```diff
@@ -69,3 +69,16 @@ rapid development so do so at your own risk. That being said, the Julia nightly
 should work using the [Developer Install](). Note that this will require you to
 be on master for the Julia and Atom packages, so things will be changing likely
 before documentation changes.
+
+## Advanced topics
+
+### Connecting to an external julia session on a remote machine
+
+Use case : local installation of Juno with a remote julia session
```

Comment:
It should be OK with the latest commit: https://github.com/JunoLab/JunoDocs.jl/pull/11/commits/8f21b877a8229c78ba38fdadc1f502161044db7c